queue/driver_mongo_test.go

```diff
@@ -54,7 +54,6 @@ func TestMongoDBOptions(t *testing.T) {
 		t.Run("FailsWithSampleSizeAndPriority", func(t *testing.T) {
 			opts := defaultMongoDBTestOptions()
 			opts.SampleSize = 100
```

Comment:
This test is no longer testing the mutually exclusive options since priority was removed, so the test case can be removed entirely.